posts/2022-08-04-jakarta-core-profile-22009-beta.adoc

```diff
@@ -185,15 +180,14 @@ One of the new features is the addition of the link:https://jakarta.ee/specifica
     </featureManager>
 ----
 
-More information regarding Jakarta Expression Language 5.0 can be  found on the 
-link:https://jakarta.ee/specifications/expression-language/5.0/[jakarta.ee site].
+For more information about Jakarta Expression Language 5.0, see the link:https://jakarta.ee/specifications/expression-language/5.0/[jakarta.ee site].
 
 [#microprofile]
 == MicroProfile OpenAPI 3.1 
 
-OpenAPI is a standardised way of documenting REST APIs in a JSON or YAML format. MicroProfile OpenAPI helps you generate and serve OpenAPI documentation for your REST applications built using JAX-RS or Jakarta restfulWS. This can be useful for developers to test out the API during development, or for people using the API in production.
```

Comment:
This change is wrong.

* **OpenAPI** is multi-language way of documenting REST APIs
* **MicroProfile OpenAPI** is a standard for Java application servers to generate OpenAPI documents for Java REST applications.

How should we reword this section to make the difference clearer?

Reply:
Eurgh, github isn't showing the right bit of code.

The original text makes a distinction between OpenAPI and MicroProfile OpenAPI. The current text has removed this distinction and is incorrect:

> OpenAPI is a standardised way of documenting REST APIs in a JSON or YAML format. It helps you generate and serve OpenAPI documentation for your REST applications that are built by using JAX-RS or Jakarta restfulWS. This can be useful for developers to test out the API during development, or for people using the API in production.

Reply:
fixed